web_src/js/utils.js

```diff
@@ -0,0 +1,3 @@
+export default function svg(name, size, staticPrefix) {
```

Comment:
Just `export function` here

Reply:
eslint got mad when I tried that, probably because it's the only function in there at the moment

Reply:
I'm surprised it warns on that. Add `import/prefer-default-export: [0]` to `rules` in `.eslintrc`.

Reply:
Ok it seems to be because of the single export (https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/prefer-default-export.md#importprefer-default-export). I think the rule is nonsense in our case, so keep it disabled gobally.